### PR TITLE
AN7581: Add uart support

### PR DESCRIPTION
**Pull request by merbanan**
Output from kernel:

1fbf0000.serial: ttyS0 at MMIO 0x1fbf0000 (irq = 14, base_baud = 115200) is a Airoha 16550

Regular console operations works fine.

### CODE REVIEW

File: target/linux/airoha/dts/en7581.dtsi [hunk 1]
```diff
@@ -146,12 +146,58 @@
...
 		uart1: serial@1fbf0000 {
-			compatible = "ns16550";
+			compatible = "airoha,en7523-uart";
 			reg = <0x0 0x1fbf0000 0x0 0x30>;
 			reg-io-width = <4>;
 			reg-shift = <2>;
 			interrupts = <GIC_SPI 18 IRQ_TYPE_LEVEL_HIGH>;
 			clock-frequency = <1843200>;
+
+			status = "okay";
```
Comment (LorenzoBianconi) [inline target/linux/airoha/dts/en7581.dtsi]: @merbanan I guess "okay" it is the default one

File: target/linux/airoha/files/arch/arm64/boot/dts/airoha/en7581.dtsi [hunk 2]
```diff
@@ -146,12 +146,58 @@
...
 		uart1: serial@1fbf0000 {
-			compatible = "ns16550";
+			compatible = "airoha,en7523-uart";
 			reg = <0x0 0x1fbf0000 0x0 0x30>;
 			reg-io-width = <4>;
 			reg-shift = <2>;
 			interrupts = <GIC_SPI 18 IRQ_TYPE_LEVEL_HIGH>;
 			clock-frequency = <1843200>;
+
+			status = "okay";
```
Comment (LorenzoBianconi) [inline target/linux/airoha/files/arch/arm64/boot/dts/airoha/en7581.dtsi]: @merbanan same here

File: target/linux/airoha/dts/en7581.dtsi (hunk 1 above)
Comment (merbanan) [reply target/linux/airoha/dts/en7581.dtsi]: Adressed.

File: target/linux/airoha/files/arch/arm64/boot/dts/airoha/en7581.dtsi (hunk 2 above)
Comment (merbanan) [reply target/linux/airoha/files/arch/arm64/boot/dts/airoha/en7581.dtsi]: Addressed.

**Pull request merged**